pkg/cli/cmd/install/kubernetes/kubernetes.go

```diff
@@ -76,6 +76,8 @@ rad install kubernetes --reinstall
 	commonflags.AddKubeContextFlagVar(cmd, &runner.KubeContext)
 	cmd.Flags().BoolVar(&runner.Reinstall, "reinstall", false, "Specify to force reinstallation of Radius")
 	cmd.Flags().StringVar(&runner.Chart, "chart", "", "Specify a file path to a helm chart to install Radius from")
+	cmd.Flags().StringVar(&runner.ContourChart, "contour-chart", "", "Specify a local file path to a helm chart to install Contour from")
```

Comment:
do these have to be a local filepath or can they be an OCI registry (e.g. on a local CR)

Reply:
local path for now